Feedback

HansMariusAndreassen opened the pull request:
Looking for review.

As this project went along I realized (learned) that I would have build things a bit differently if I was to do it over again. More modular with separating things more, but I am happy with how I am able to use the same functions for different pages.
Practice makes perfect I guess!

HansMariusAndreassen committed:
setting up branch for dev

HansMariusAndreassen committed:
Fetching profile and post filter

HansMariusAndreassen committed:
branch fix

HansMariusAndreassen committed:
update to the login and fetch

Fetching profile set up

HansMariusAndreassen committed:
implemented fetch and render on feed

fetch, follow/unfollow and filtering. create posts and posts

HansMariusAndreassen committed:
update login scss

side scroll caused by gutter. set a new override to max-1 100%

HansMariusAndreassen committed:
index update

added functionality to disable nav links when not logged in.

HansMariusAndreassen committed:
fetch and display posts

added fetch and display functions. refactored some of the functions

HansMariusAndreassen committed:
Merge branch 'js2' into Fetching-posts

HansMariusAndreassen committed:
Merge pull request #2 from Spookyrumble/Fetching-posts

developing feed

HansMariusAndreassen committed:
Update index.html

HansMariusAndreassen committed:
Merge branch 'Fetching-posts' of https://github.com/Spookyrumble/css-frameworks-ca into Fetching-posts

HansMariusAndreassen committed:
Update index.js

HansMariusAndreassen committed:
started filter and search functions

HansMariusAndreassen committed:
bug fix follow/unfollow function

HansMariusAndreassen committed:
fixed filter and search

also setting up for edit and delete

HansMariusAndreassen committed:
fixed delete post link/function.

setting up for edit

HansMariusAndreassen committed:
filter and search updates

HansMariusAndreassen committed:
jsDocs

HansMariusAndreassen committed:
Merge pull request #3 from Spookyrumble/Fetching-posts

Fetching posts + filter, search, edit, delete

HansMariusAndreassen committed:
removed dist from gitignore

HansMariusAndreassen committed:
Update index.scss

HansMariusAndreassen committed:
Update postHandler.mjs

HansMariusAndreassen committed:
Update index.js

HansMariusAndreassen committed:
Update index.js

fixed register new user

HansMariusAndreassen committed:
Update index.js

updated jsdocs

HansMariusAndreassen committed:
Update index.js

login fix

HansMariusAndreassen committed:
npm deployment fix

HansMariusAndreassen committed:
Update index.html

HansMariusAndreassen committed:
comments mostly in place.

Still some kinks to work out

HansMariusAndreassen committed:
added update user media functionality

HansMariusAndreassen committed:
Merge pull request #4 from Spookyrumble/developing-comments-and-reactions

Developing comments and reactions

HansMariusAndreassen committed:
update styling

added "active" on nav items. Increased spacing on nav items. Adjusted button sizes and placement. fixed nav logo centering on smaller screens.

HansMariusAndreassen committed:
Merge pull request #6 from Spookyrumble/debugging-deployment

update styling

HansMariusAndreassen committed:
update button margin

HansMariusAndreassen committed:
styling update

HansMariusAndreassen committed:
update styling

HansMariusAndreassen committed:
follow unfollow fix

sulenchy reviewed (approved):
@Spookyrumble This is a good job. You have all the functionalities already. I am impressed. However, there is one more thing you missed. An authenticated user can still access the login and register page. Even though this is not explicitly stated in the requirement, it is expected. A logged-in user should not access the login and register page unless he/she signs out.

HansMariusAndreassen committed:
update search

HansMariusAndreassen committed:
updated styling and security

made nav sticky.
Added window.replace logic to hinder access to base index if already logged in.

Spookyrumble commented:
Updated some styles to accommodate feedback.
Also added logic to deny access to /index if already logged in.

HansMariusAndreassen committed:
space adjustments and user logged in check for index

HansMariusAndreassen committed:
Update README.md

HansMariusAndreassen committed:
set new comment rules

to prevent spam feeds, I set the function to nly display 4 comments and added a button to display the rest if needed.

HansMariusAndreassen committed:
WAVE updates

missing labels on "add comment" textarea and a low contrast concern on navigation.

HansMariusAndreassen committed:
Update README.md

HansMariusAndreassen committed:
fixed header spacing on index

HansMariusAndreassen committed:
implemented feedback changes

removed disabled nav from reg/login.
made create user form to automatically log in and redirect to /feed.
Adjusted H1 on index.
removed loader on profile. If we weren't so close to deadline I would have it not display on users without posts. Removed it for now.

HansMariusAndreassen committed:
Update README.md

HansMariusAndreassen committed:
fixed profile feed.

Loader removal caused crash

HansMariusAndreassen committed:
burger fix

low contrast.
